packages/block-library/src/breadcrumbs/index.php

```diff
@@ -34,11 +34,8 @@ function render_block_core_breadcrumbs( $attributes, $content, $block ) {
 		return '';
 	}
 
-	$separator        = isset( $attributes['separator'] ) ? $attributes['separator'] : '/';
```

Comment:
I think we might still want to check whether `separator` is set to avoid accidental PHP warnings.

Reply:
Can we get to a point where the `separator` key will not be set?

Reply:
It's not a blocker; we can follow up later as needed.